README.md

```diff
@@ -82,48 +83,86 @@ Shell return code will be non-zero if at least one error was found.
 3. If that doesn't work or if you're on Windows, add environment variable `DART_VM_OPTIONS` with value `--root-certs-file=<cert_file>`.
 
 After doing this, `dart pub get` should be able to download dependencies successfully.
 
+#### Clearing Caches
+
+In rare cases, you may encounter various build issues usually caused by previously cached artifacts and/or packages. To clear all caches and re-download the dependencies, do the following steps:
+
+1. Remove the `.dart_tool` directory and the `pubspec.lock` file from the project root directory.
+2. Run `dart pub cache clean --force`.
+3. Run `dart pub get`.
+
 ### Drag-n-Drop Web Tool
-To build a drag-n-drop online validation tool (as hosted [here](https://github.khronos.org/glTF-Validator/)), follow these steps after installation:
+
+To build the drag-n-drop online validation tool (as hosted [here](https://github.khronos.org/glTF-Validator/)), follow these steps after installation:
+
 1. Run `dart run grinder web`.
 2. All needed files will be written to `build/web` directory.
 
+To build and package the drag-n-drop online validation tool, follow these steps after installation:
+
+1. Run `dart run grinder web-archive`.
+2. An archive will be written to the `build/gltf_validator-VERSION-web.zip` file.
+
 ### CLI tool
+
+To build the CLI tool, follow these steps after installation:
+
 1. Run `dart run grinder exe`.
-2. Native executable file will be written to `build/bin/gltf_validator` or `build/bin/gltf_validator.exe`.
+2. Native executable file will be written to `build/bin/gltf_validator` (Linux and macOS) or `build/bin/gltf_validator.exe` (Windows).
+
+To build and package the CLI tool, follow these steps after installation:
+
+1. Run `dart run grinder exe-archive`.
+2. An archive will be written to the `build/gltf_validator-VERSION-PLATFORM.tar.xz` (Linux and macOS) or `build/gltf_validator-VERSION-PLATFORM.zip` (Windows) file.
 
 ### NPM Package
+
 To build an npm package for use in Node.js environment, follow these steps after installation:
+
 1. Run `dart run grinder npm`.
 2. `gltf-validator` npm package will be written to `build/node`.
 
 Refer to the [npm package documentation](https://www.npmjs.com/package/gltf-validator) for additional information.
 
 #### Publishing
+
 To publish an npm package, follow these steps after installation:
+
 1. Run `dart run grinder npm-publish`.
 2. `gltf-validator` npm package will be built to `build/node` and published to npm registry using `npm publish`.
 
-### Post-Build Steps
-1. To auto-format all source files, from the root folder run: `dart format --fix .`
-2. To run the tests, use: `dart test`
-
 ### Validation Issues List
+
 To generate [ISSUES.md](ISSUES.md), follow these steps after installation:
+
 1. Run `dart run grinder issues`.
 2. `ISSUES.md` file will be written to the repo root.
+
+## Contributing
+
+Before making a pull request do the following steps:
+
+1. Run `dart format .` (note the dot) to format all source files.
```

Comment:
We lost a parameter here, `dart format --fix .`  Was that intentional?

Reply:
Yes, that option should not be used in regular workflow.